app/src/renderer/components/governance/PageGovernance.vue

```diff
@@ -66,6 +67,12 @@ export default {
     // TODO: get min deposit denom from gov params
     ...mapGetters([`proposals`, `filters`, `bondingDenom`, `connected`])
   },
+  mounted() {
```

Comment:
how about putting this logic into TmPage?

Reply:
i know 😞... but i want to get rid of TmPage and also only should run where tabs are present (not on wallet, transactions, or preferences). also, after we bring tendermint-ui into voyager this kind of thing will be easier / more obvious.

Reply:
there's an issue for it on #1391